doc/src/modules/solvers/ode.rst

```diff
@@ -156,7 +156,7 @@ linear_coefficients
 
 separable_reduced
 ^^^^^^^^^^^^^^^^^
-.. autofunction:: sympy.solvers.ode.ode::ode_separable_reduced
+.. autofunction:: sympy.solvers.ode.single::SeparableReduced
 
```

Comment:
I had a doubt as it was inherited from `Separable` class, In `ode.rst` when I wrote :members: it gave an error can yu explain the reason?